General/Refactorings.md

```diff
@@ -0,0 +1,528 @@
+# Refactoring in Pharo 8.0
```

Comment:
Here you specify that this page is for Pharo 8.0 so no need for the Pharo 8.0 in the README.md I think.